src/ORM/DB.php

```diff
@@ -109,6 +126,124 @@ public static function get_conn($name = 'default')
         return null;
     }
 
+    /**
+     * Only use the primary database connection for the rest of the current request
+     *
+     * @internal
+     */
+    private static bool $mustUsePrimary = false;
```

Comment:
Move this and other properties to be above all methods per https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#class-member-ordering

Reply:
Updated